lua/config/defaults.lua

```diff
@@ -578,6 +578,12 @@ lvim.lang = {
             end,
           },
         },
+        get_language_id = function(filetype)
+          if vim.tbl_contains({ "json" }, filetype) then
```

Comment:
This seems like a complicated way to write `if filetype == "json"`

Does this mean that all json files will be considered as jsonc files ?

Reply:
There's already a upstream merge request for this.

https://github.com/neovim/nvim-lspconfig/pull/1195

Reply:
That's good news, it means we can remove it from here then